oxidation/libparsec/crates/crypto/Cargo.toml

```diff
@@ -32,6 +32,10 @@ sha2 = { version = "0.10.6" }
 x25519-dalek = { version = "2.0.0-pre.1", optional = true }
 xsalsa20poly1305 = { version = "0.9.0", optional = true }
 
+openssl = { version = "0.10" }
+openssl-sys = "0.9"
+zeroize = { version = "1", features = ["alloc"] }
```

Comment:
This need to be put behind a feature, since we can't include this in the wasm build

Reply:
Fixed in d59e7dc27e4c62dbff53fbed708a69dcea2e5a39